rocAL/rocAL/source/meta_data/tf_meta_data_reader_detection.cpp

```diff
@@ -177,10 +177,10 @@ void TFMetaDataReaderDetection::read_record(std::ifstream &file_contents, uint f
       bbox_ymin = sf_ymin.float_list().value()[i];
       bbox_xmax = sf_xmax.float_list().value()[i];
       bbox_ymax = sf_ymax.float_list().value()[i];
-      box.l = bbox_xmin ;
-      box.r = bbox_xmax ;
-      box.t = bbox_ymin ;
-      box.b = bbox_ymax ;
+      box.l = bbox_xmin * image_width;
```

Comment:
please combine this math with the previous 4 lines

Reply:
Combined this in the latest commit